src/coreclr/nativeaot/BuildIntegration/BuildIntegration.proj

```diff
@@ -11,6 +11,11 @@
 
   <Target Name="Build">
     <Copy SourceFiles="@(Content)" DestinationFolder="$(OutputPath)" />
+
+    <!-- Create breadcrumb to disable dynamic linking of release crt for debug runtime -->
+    <WriteLinesToFile
+      File="$(RuntimeBinDir)/aotsdk/debugucrt.txt"
```

Comment:
It isn't obvious that `$(RuntimeBinDir)/aotsdk` is the same as `IlcSdkPath`, is that the case here?

Reply:
`$(RuntimeBinDir)/aotsdk` is an intermediate directory in the `dotnet/runtime` repo build. `IlcSdkPath` is a directory inside the NativeAOT nuget package. `IlcSdkPath` in the nuget package gets populated from `$(RuntimeBinDir)/aotsdk`. They are not same directories.